sdk/rust/src/query/execute.rs

```diff
@@ -35,44 +35,38 @@ use crate::{
     Error,
     FromProtobuf,
     Hbar,
+    LedgerId,
     Query,
     Status,
     TransactionId,
 };
 
-/// Describes a specific query that can be executed on the Hedera network.
```

Comment:
please do leave the doc comments in, it's internal documentation but it's still useful (shows up in IDEs while working on the project, `cargo doc --document-private-items` exists as well for internal development